conda-recipes/ray-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "ray" %}
+{% set version = "0.7.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ray-project/ray.git
+  git_rev: ray-{{ version }}
+  patches:
+    - redis-bazel-build.patch
+    
+build:
+  noarch: python
```

Comment:
I think this package includes compiled host code, so maybe shouldn't be `noarch`.

Reply:
ok I removed the "noarch: python"